src/ncp/ncp_base_ftd.cpp

```diff
@@ -1508,6 +1508,34 @@ otError NcpBase::InfraIfSendIcmp6Nd(uint32_t            aInfraIfIndex,
 
 #endif // OPENTHREAD_CONFIG_NCP_INFRA_IF_ENABLE && OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
 
+#if OPENTHREAD_CONFIG_SRP_SERVER_ENABLE
+template <> otError NcpBase::HandlePropertySet<SPINEL_PROP_SRP_SERVER_ENABLED>(void)
+{
+    otError error = OT_ERROR_NONE;
+    bool    enable;
+
+    SuccessOrExit(error = mDecoder.ReadBool(enable));
+    otSrpServerSetEnabled(mInstance, enable);
+
+exit:
+    return error;
+}
```

Comment:
Should we add `HandlePropertyGet` for these?

Reply:
Sounds good! Under current design the PropertyGet for the two is not used. But let's add them to avoid changing the spinel.h later when they are needed. I've added them.